core/com.b2international.snowowl.datastore.server.tests/src/com/b2international/snowowl/datastore/internal/branch/MockInternalCDOBranchManager.java

```diff
@@ -48,7 +48,7 @@
 	
 	public MockInternalCDOBranchManager(CDOTimeProvider clock) {
 		this.clock = clock;
-		when(delegate.getTimeProvider()).thenReturn(clock);
+//		when(delegate.getTimeProvider()).thenReturn(clock);
```

Comment:
What is the reason for these commented lines?

Reply:
We can remove those, the new Mockito complains about them that they are unused stubs and should be removed.

Reply:
👍